### ISSUE TITLE
"nice error message" when running tests

### ISSUE THREAD

**Issue by DanielGrobelny** [webpack/webpack-dev-middleware#134]
Hi there,

since 1.8.2 we get an error message when running our tests with karma.
The application itself works like a charm ... any hints?

```
npm run test

superawesome-secret-application@0.0.1 test C:\Users\myusername\dev\superawesome-secret-application
cross-env NODE_ENV=test karma start

26 09 2016 17:22:36.751:ERROR [preprocess]: Can not load "webpack"!
  Error: 'output.path' needs to be an absolute path or '/'.
    at new module.exports (C:\Users\myusername\dev\superawesome-secret-application\node_modules\webpack-dev-middleware\middleware.js:60:9)
    at new Plugin (C:\Users\myusername\dev\superawesome-secret-application\node_modules\karma-webpack\lib\karma-webpack.js:123:38)
    at invoke (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:75:15)
    at Array.instantiate (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:59:20)
    at get (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:48:43)
    at C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:71:14
    at Array.map (native)
    at Array.invoke (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:70:31)
    at get (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:48:43)
    at instantiatePreprocessor (C:\Users\myusername\dev\superawesome-secret-application\node_modules\karma\lib\preprocessor.js:55:20)
    at Array.forEach (native)
    at createPreprocessor (C:\Users\myusername\dev\superawesome-secret-application\node_modules\karma\lib\preprocessor.js:74:20)
    at Array.invoke (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:75:15)
    at get (C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:48:43)
    at C:\Users\myusername\dev\superawesome-secret-application\node_modules\di\lib\injector.js:71:14
    at Array.map (native)
```

Thanks and kind regards,
Daniel

**Comment by SpaceK33z** [comment]
Yeah I fixed this in 38ff513cf6a5e5a909134db0fc6614a1831fbdd1, but that hasn't been released yet.

**Comment by SpaceK33z** [comment]
Released `1.8.3`. Let me know if you still have problems.
